components/script/dom/event.rs

```diff
@@ -433,6 +433,15 @@ impl EventMethods for Event {
         self.current_target.get()
     }
 
+    // https://dom.spec.whatwg.org/#dom-event-composedpath
```

Comment:
nit: `///`

Reply:
should I use `///` for all other functions to keep it consistent ?

Reply:
If they are unrelated, it would be better to do it separately. Not very important.